packages/dx-scheduler-core/src/plugins/scheduler-core/computeds.ts

```diff
@@ -17,3 +18,22 @@ export const appointments: PureComputed<
   },
   dataItem: appointment,
 }));
+
+export const dateTimeFormatComputed: any = (locale: string | string[]) => {
+  const cache = new Map();
+
+  const formatter = (nextDate: Date | string | number, nextOptions: Intl.DateTimeFormatOptions) => {
+    if (nextDate === undefined) return;
+    const date = (typeof nextDate) === 'object' ? nextDate : new Date(nextDate);
+    const key = JSON.stringify(nextOptions);
```

Comment:
`JSON.stringify` is not zero-cost operation.
Is it that much faster than `new Intl.DateTimeFormat`?

Why is `JSON.stringify` used?
For example, `{ a, b }` and `{ b, a }` would be different.
May be `nextOptions` instance itself can be used as key?